frontend/src/libs/components/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,13 @@
+import 'react-tooltip/dist/react-tooltip.css';
+
+import { type FC } from 'react';
+
+import { ReactTooltip } from '~/libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+const Tooltip: FC = () => {
+  return <ReactTooltip id="main-tooltip" className={styles.tooltip} />;
+};
+
+export { Tooltip };
```

Comment:
Also create here separate component for wrapping tooltip logic
Something like this
```
type ElementWithTooltipProps = {
	className?: string;
	elementType: keyof ReactHTML;
	placement?: 'bottom' | 'top' | 'left' | 'right';
	tooltipId?: TooltipId;
	tooltipContent: string | ReactElement;
};

const ElementWithTooltip: React.FC<ElementWithTooltipProps> = ({
	children,
	className,
	elementType,
	placement = 'top',
	tooltipId = DataTooltipId.MAIN_TOOLTIP,
	tooltipContent
}) => {
	const isStringContent = typeof tooltipContent === 'string';
	const contentDataAttribute = isStringContent ? 'data-tooltip-content' : 'data-tooltip-html';
	const adaptedTooltipContent = isStringContent ? tooltipContent : renderToStaticMarkup(tooltipContent);

	return createElement(
		elementType,
		{
			'data-tooltip-id': tooltipId,
			'data-tooltip-place': placement,
			className,
			[contentDataAttribute]: adaptedTooltipContent
		},
		children
	);
};
```
And another component to inherit `ElementWithTooltip`
```
type Props = Omit<ElementWithTooltipProps, 'elementType'>;

const BlockWithTooltip: React.FC<Props> = ({ children, ...restProps }) => {
	return (
		<ElementWithTooltip {...restProps} elementType="div">
			{children}
		</ElementWithTooltip>
	);
};
```
And this way you may just wrap needed element with `BlockWithTooltip` and pass tooltipContent as prop

Reply:
done - [86f2cad](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/156/commits/86f2cad1c05b7e6e79fd2815ddb60593a5859b04)